lfs/batcher.go

```diff
@@ -9,6 +11,7 @@ package lfs
 // When a timeout, Flush(), or Exit() occurs, the group may be smaller than the
 // batch size.
 type Batcher struct {
+	exited     uint32
```

Comment:
Even though this field is private, maybe a quick note here about what 1 and 0 mean in the context of this member?